maps/torch/torch6_bridge.dmm

```diff
@@ -17982,9 +17994,19 @@
 	icon_state = "chair_preview"
 	},
 /obj/effect/floor_decal/techfloor{
-	dir = 8;
-	icon_state = "techfloor_edges"
+	dir = 8
 	},
+/obj/structure/closet/secure_closet/wall{
+	anchored = 1;
+	desc = "It's a card-locked storage unit. ONLY FOR USE IN EXTREME EMERGENCY.";
+	name = "DELTA LEVEL SAFE";
+	pixel_x = -25;
+	req_access = list("ACCESS_VAULT");
+	wall_mounted = 1
+	},
+/obj/item/weapon/disk/nuclear,
+/obj/item/weapon/pinpointer/advpinpointer,
+/obj/item/weapon/folder/envelope/nuke_instructions,
```

Comment:
Just checking were does the Nuke disk spawn before this? Like on a table or something?

Reply:
No, in a closed locker